hack/prebuild/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "packet_project" "libvirt_actuator" {
```

Comment:
Does this create a project for every build? We probably want a single re-used project. That will let us manage permissions to for it.